### ISSUE TITLE
Add "Always open in this container" contextual menu option to built-in Container feature

### ISSUE THREAD

**Issue by practik** [mozilla/multi-account-containers#1007]
This bug is not for the Multi-Account Containers add-on; it's for the built-in version of Containers in FF 57. This seems like the wrong place to report this, but I was sent here from Bugzilla.

I've read that earlier versions of Containers (Nightly feature, Test Pilot experiment) included an "Always open in this container" contextual menu item, as does the Multi-Account Containers add-on. However, that menu item seems to be missing from the current built-in version of Containers – or at least I've never seen it, and I've right-clicked on a bunch of webpages in hopes of using it.

At the moment I'm using the Containerise add-on to assign domains to containers, but it would be nice if the feature were natively included. Is that in the works? Or am I just missing it somehow?

I'm running FF 57.0 on macOS 10.12. No experimental features. I can list my extensions if you like, but I've tested in Safe Mode as well and I can't see the menu item then either.

**Comment by jonathanKingston** [comment]
> This seems like the wrong place to report this, but I was sent here from Bugzilla.
Hmm this is odd, do you have a bug id?

For this feature:
1. Install multi-account containers
2. open a container tab
3. Right click and get the context menu for Multi-account containers
4. Under this menu should be "Always open in this container"

It is also accessible from the multi-account containers menu in the toolbar as a checkbox.

Can you confirm this is the case?

**Comment by practik** [comment]
> do you have a bug id?

https://bugzilla.mozilla.org/show_bug.cgi?id=1420293

> Can you confirm this is the case?

No, because I don't have Multi-Account Containers installed. As I said, I'm asking for the menu item to be added to Firefox's built-in Container feature. Apologies if this is not the place for that.

**Comment by grahamperrin** [comment]
https://bugzilla.mozilla.org/show_bug.cgi?id=1420293#c1

> Bugs in the container functionality are being handled on GitHub. … 

I wasn't aware of that. So, if there are to be related changes to code for Firefox, where is the code developed then reviewed (before being committed)?

**Comment by practik** [comment]
> if there are to be related changes to code for Firefox, where is the code developed then reviewed (before being committed)?

Exactly. Maybe Kohei gave me bad information? I'll be curious to see what, if anything, Jorge says.
